### PR TITLE
Add notes to qualify lack of Large Object support.

### PR DESCRIPTION
**Pull request by dyozie**
Adding note around current documentation that suggests large object support akin to postgresql.

### CODE REVIEW

**Comment by dyozie** [comment]
@Eulerizeit @hlinnaka - Can you clarify what we need to say about (lack of) large object support in GPDB, specifically for the questions @litzell raises above?  Thanks!

**Comment by Eulerizeit** [comment]
We don't support large objects because we couldn't figure out an
intelligent way to deal with motions. Given replicated tables this might be
something to look into for 7 but personally I'm not thrilled with them as a
feature, they seem like second class citizens.

I don't know what we have done with the code wrt disallowing their use.
I'll check into that.

Rob

**Comment by dyozie** [comment]
Thanks for the info. We'll keep the current warning re: non-support in the
areas where large object syntax isn't hidden. For things like ALTER LARGE
OBJECT we'll omit the docs entirely.

File: gpdb-doc/dita/ref_guide/system_catalogs/pg_largeobject.xml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE topic
   PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
-<topic id="topic1" xml:lang="en"><title id="gt143896">pg_largeobject</title><body><p>The <codeph>pg_largeobject</codeph> system catalog table holds the data
+<topic id="topic1" xml:lang="en"><title id="gt143896">pg_largeobject</title><body>
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/system_catalogs/pg_largeobject.xml]: are we going to remove this system catalog table?

File: gpdb-doc/dita/utility_guide/client_utilities/pg_restore.xml [hunk 2]
```diff
@@ -275,11 +275,6 @@
         disable triggers on user tables before inserting the data then emits commands to re-enable
         them after the data has been inserted. If the restore is stopped in the middle, the system
         catalogs may be left in the wrong state.</p>
-      <p><codeph>pg_restore</codeph> cannot restore large objects selectively,
```
Comment (lisakowen) [inline gpdb-doc/dita/utility_guide/client_utilities/pg_restore.xml]: we are removing this large object reference on the pg_restore page, but on the COMMENT, pg_dump, and psql ref pages we are keeping the large object reference and then adding the note.  should we be consistent?

Comment (dyozie) [reply gpdb-doc/dita/utility_guide/client_utilities/pg_restore.xml]: I think there's a difference in calling out functionality that we don't actually support (as with pg_restore) vs actual syntax that will show up via help (with COMMENT, etc.).  If the syntax is easily visible via help output, I think we need to call out things that aren't expected to work.

File: gpdb-doc/dita/ref_guide/system_catalogs/pg_largeobject.xml (hunk 1 above)
Comment (dyozie) [reply gpdb-doc/dita/ref_guide/system_catalogs/pg_largeobject.xml]: I'm open to comments here, but as in the case of syntax it's hard to hide the presence of catalog tables.  I think we should at least list it and call out that it isn't used.  Maybe we can do that but get rid of the column descriptions?

**Pull request merged**